scripts/chain-prep.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#set -eux
+killall node
+ganache-cli -u 0 -u 1 -u 2 -m 'cook mango twist then skin sort option civil have still rather guilt' &
+git clone https://github.com/althea-mesh/simple-bidirectional-erc20-channel contract
+pushd contract 
+npm install .
+truffle compile
+addresses="$(truffle migrate --verbose)"
+export CHANNEL_ADDRESS="$(echo $addresses | grep -Eo ': 0x[0-9a-f]{40}'| cut -d$'\n' -f3)"
+export TOKEN_ADDRESS="$(echo $addresses | grep -Eo ': 0x[0-9a-f]{40}'| cut -d$'\n' -f4)"
```

Comment:
I'm playing with deployed contract and noticed this is probably not necessary to pass. Do we need this address and if so what would be its purpose in guac?

Reply:
eventually we want gauc to work with erc20 tokens and at that point we probably need the token contract address for various things, not really needed right now.